packages/modules/web_themes/koala/source/src/components/BatteryInformation.vue

```diff
@@ -18,9 +15,11 @@ import BatteryCard from 'src/components/BatteryCard.vue';
 
 const mqttStore = useMqttStore();
 
-const batteryIds = computed(() => mqttStore.batteryIds);
-
-const showBatteryOverview = computed(() => {
-  return mqttStore.batteryIds.length > 1;
+const batteryIds = computed(() => {
+  let ids = mqttStore.batteryIds;
+  if (ids.length > 1) {
```

Comment:
Um zu vermeiden, dass die Übersichtskarte erneut eingefügt wird (z. B. beim Hin- und Her wechseln zwischen Tabs), können wir zuerst auf -1 prüfen.

```suggestion
if (ids.length > 1 && !ids.includes(-1)) {